$NetBSD: patch-meson.build,v 1.1 2019/06/03 10:47:20 prlw1 Exp $

Don't build tests as they depend on many other packages.

--- meson.build.orig	2019-04-07 13:42:45.000000000 +0000
+++ meson.build
@@ -166,7 +166,6 @@ subdir('giscanner')
 subdir('gir')
 subdir('examples')
 subdir('docs')
-subdir('tests')
 
 install_data('Makefile.introspection', install_dir: join_paths(get_option('datadir'), 'gobject-introspection-1.0'))
 install_data('m4/introspection.m4', install_dir: join_paths(get_option('datadir'), 'aclocal'))
